lib/laa_crime_forms_common/pricing/nsm/calculators/cost_summary.rb

```diff
@@ -22,12 +22,18 @@ def call
               disbursements: disbursements_summary_row,
               travel: travel_summary_row,
               waiting: waiting_summary_row,
+              additional_fees: additional_fees_row,
```

Comment:
I'm really not sure about this. Additional fees are _part_ of profit costs; I think it's misleading to double-count them by having them both in the profit costs section and on their own.

Reply:
If in future we have additional non-profit-cost fees, then they should be treated separately, because they would belong in different groups of the cost summary.

Reply:
yeah - I was deliberating about this, the reason I kept it in the cost_summary was to prevent running a calculation twice as I know in the UI we need additional cost totals AND need to add at least some (for now all) additional costs to the profit costs, but you're right semantically it doesn't make sense to be here

Reply:
So looking at Figma, you need to show a breakdown of the additional fees. So ideally you'd have the calculator return a fee breakdown that lists all the fees by name (and yes, there is only one of them maximum, so it's probably just a hash with a `youth_court` key and no other keys). I still think you need this outside the cost summary, because it's not something that would go in the cost summary table.
<img width="427" alt="Screenshot 2024-11-27 at 14 51 09" src="https://github.com/user-attachments/assets/1f931051-62b3-4c77-9762-74d9bc74126f">

Reply:
> agreed, this has been updated now